hedera-node/src/test/java/com/hedera/test/factories/txns/TokenCreateFactory.java

```diff
@@ -50,13 +54,16 @@ public TokenCreateFactory autoRenew(AccountID account) {
 		return this;
 	}
 
-	public TokenCreateFactory missingTreasury() {
-		omitTreasury = true;
+	public TokenCreateFactory missingAdmin() {
+		omitAdmin = true;
 		return this;
 	}
 
-	public TokenCreateFactory missingAdmin() {
-		omitAdmin = true;
+	public TokenCreateFactory plusCustomFee(CustomFee customFee) {
+		if (customFees == null) {
+			customFees = new ArrayList<>();
+		}
```

Comment:
should initialize `private List<CustomFee> customFees = new ArrayList<>();` and remove this if